# Configurações para o publicador de TF do frame 'odom'
# Este arquivo define os parâmetros para a transformação entre 'odom' e 'base_footprint'
#
# odom_tf_publisher:
#   ros__parameters:
#     # Posição inicial do robô no frame odom (metros)
#     initial_x: 0.0
#     initial_y: 0.0
#     initial_z: 0.0
#     
#     # Orientação inicial do robô (quaternion)
#     initial_qx: 0.0
#     initial_qy: 0.0
#     initial_qz: 0.0
#     initial_qw: 1.0
#     
#     # Taxa de publicação (Hz)
#     publish_rate: 50.0
#     
#     # Frame de origem (pai)
#     parent_frame: "odom"
#     
#     # Frame de destino (filho) 
#     child_frame: "base_footprint"

odom_tf_publisher:
  ros__parameters:
    initial_x: 0.0
    initial_y: 0.0
    initial_z: 0.0
    initial_qx: 0.0
    initial_qy: 0.0
    initial_qz: 0.0
    initial_qw: 1.0
    publish_rate: 50.0
    parent_frame: "odom"
    child_frame: "base_footprint"
